.functional_ci_setup.py

```diff
@@ -92,33 +92,36 @@ def get_ocsci_conf():
             region=env['AWS_REGION'],
             base_domain=env['AWS_DOMAIN'],
             worker_instance_type='m5.4xlarge',
+            cluster_namespace="openshift-storage",
+
         ),
     )
     if env.get("DOWNSTREAM") == "true":
         conf_obj['REPORTING'] = dict(us_ds='DS')
     if env.get("OCS_OPERATOR_DEPLOYMENT") == "true":
         conf_obj['DEPLOYMENT'] = dict(
-            ocs_operator_deployment=True,
             ocs_operator_image=env['OCS_OPERATOR_IMAGE'],
-            ocs_registry_image=env['OCS_REGISTRY_IMAGE'],
+            ocs_operator_deployment=True,
```

Comment:
@zmc not sure if you are naming clusters the same before first dash, but will be worth if you will add also this parameter:
force_deploy_multiple_clusters: True

It's recent changed introduced here:
https://github.com/red-hat-storage/ocs-ci/pull/780/files

For preventing ppl to create new clusters if they already have one.